port_drayage_plugin/include/port_drayage_plugin/port_drayage_worker.h

```diff
@@ -29,17 +31,17 @@
 #include "port_drayage_plugin/port_drayage_state_machine.h"
 
 namespace port_drayage_plugin
-{
+{    
     /**
      * Convenience struct for storing all data contained in a received MobilityOperation message's
      * strategy_params field with strategy "carma/port_drayage"
      */
     struct PortDrayageMobilityOperationMsg
     {
         boost::optional<std::string> cargo_id;
-        std::string operation;
+        std::string operation = "";
         PortDrayageEvent port_drayage_event_type; // PortDrayageEvent associated with this message
-        bool has_cargo; // Flag to indicate whether vehicle has cargo during this action
+        bool has_cargo = false; // Flag to indicate whether vehicle has cargo during this action
```

Comment:
Is `has_cargo` a necessary data element to track if we have `cargo_id` as an optional element? Is there ever a case where `cargo_id` exists and `has_cargo` would be false or vice-versa?

Reply:
When I initially implemented this, I thought it was cleaner to have an explicit `has_cargo` flag for this. I completely agree though that it is not necessary to track both, and a better solution is to only track the `_cargo_id` element (there is never a case where `_cargo_id` and `has_cargo` conflict with each other). I have removed the `has_cargo` element.

Reply:
Good catch. Paul Bourelly and I had updated the design to remove the `has_cargo` flag from messages sent from V2XHub to carma-platform, so I have removed it.